docs/extend/plugins_authorization.md

```diff
@@ -58,7 +58,7 @@ respectively.
 
 ## Default user authorization mechanism
 
-If TLS is enabled in the [Docker daemon](../security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
+If TLS is enabled in the [Docker daemon](https://docs.docker.com/engine/security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
```

Comment:
yes, at the beginning I change all ".md" to "/", but when I meet the following:
"../tutorials/dockervolumes.md#mount-a-host-directory-as-a-data-volume"
should it be changed to "https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-directory-as-a-data-volume" or "https://docs.docker.com/engine/tutorials/dockervolumes/"?
and another one:
"../swarm/images/services-diagram.png)"
should it be changed to "https://docs.docker.com/engine/swarm/images/services-diagram/), which is a invalid address, how about your opinions, thanks :)